### ISSUE TITLE
getting following error while running arxiv_cs_abstract

### ISSUE THREAD

**Issue by anamika1302** [chrisdonahue/ilm#12]
![Screenshot from 2022-09-18 18-34-08](https://user-images.githubusercontent.com/11577703/190930749-15c41946-497d-4f34-81b5-7b3b6a88da94.png)

**Comment by paulyoussef** [comment]
You are probably using a new version of the Transformers library. Replacing  `eval_logits, _ = model(eval_inputs)` with `eval_logits = model(eval_inputs).logits` worked out for me.
